.github/workflows/lint-python.yml

```diff
@@ -84,7 +84,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ inputs.python-version }}
```

Comment:
don't forget the `cache: "pip|poetry|pipenv"`

Reply:
These ones don't need the dependencies cache, we're not installing _all_ the dependencies, we just need python to be able to run the linting tool.